fix(noUselessFragments): check Fragment when it under JSX Element or Condition Expr (#4882)
biomejs/biome: .changeset/fix_fragament_4751.md

---
biome_analyze: major
---

# Fix Fragment when it under JSXElement or JS Condition expr

> Fix [#4751](https://github.com/biomejs/biome/issues/4751), check Fragement when it under JSXElement or JS Condition expr, for example:

Reviewer: ```suggestion
Fix [#4751](https://github.com/biomejs/biome/issues/4751) by checking fragments inside `JSXElement` and conditional expressions. For example:
```

Author: done
